src/main/webapp/pages/projects/project_new.html

```diff
@@ -94,15 +94,22 @@ <h2 class="panel-title" th:text="#{projects.create.heading}">Create a New Projec
                                         <li th:each="sample : ${disallowedSamples}" th:text="${sample.sampleName}">_sample_</li>
                                     </ul>
                                 </div>
-                            </th:block>
-                            
+
                                 <div class="checkbox">
-                                    <label for="assemble">
-                                        <input type="checkbox" id="assemble" name="assembleUploads" value="true"/>
-                                        <span th:text="#{projects.create.settings.assemble}">_Assemble_</span>
+                                    <label for="lockSamples">
+                                        <input type="checkbox" id="lockSamples" name="lockSamples" value="false" th:disabled="${allSamples} == 0" />
```

Comment:
When I go to `/projects/new` both the "Add Samples..." and "Lock sample..." boxes are unchecked.  Shouldn't the lock one be disable at this point?

![image](https://user-images.githubusercontent.com/8951708/55966795-b1123900-5c3e-11e9-8f80-c8dc692f5324.png)

Reply:
Fixed in 9e258b0be30f71fb335b7baf9d4591ba3e44601b